public/scripts/app.js

```diff
@@ -47,66 +39,36 @@ $('ul.nav li').click(function(){
          `
        );
      }
-   }
-   else if(($(this).context.textContent)==="My Work")
-   {
-     //projects section
-
+    } else if(($(this).context.textContent)==="My Work") {
      $("#addProject").fadeIn();
      $("#addVacation").fadeOut();
-
-     console.log("Projects section");
      $.ajax({
        method:"GET",
-       url:'/api/profile/projects',
```

Comment:
You don't need this API endpoint – it is giving you the same exact response as `/api/profile`, so use that one instead.